app/adapters/alma_adapter/status.rb

```diff
@@ -14,11 +14,21 @@ def initialize(bib:, holding:, aeon: false)
   end
 
   def to_s
-    return "On-site Access" if aeon
+    return "On-site Access" if on_site_holding?
     return "Some items not available" if holding["availability"] == "check_holdings"
     return holding["availability"].titlecase if holding["availability"]
 
     # For electronic holdings
     return holding["activation_status"].titlecase if holding["activation_status"]
   end
+
+  def on_site_holding?
+    return true if aeon
+    on_site_locations.include? "#{holding['library_code']}$#{holding['location_code']}"
+  end
+
+  # Holdings in these location are for on-site use only
+  def on_site_locations
+    ["lewis$map", "lewis$maplf", "lewis$maplref", "lewis$mapmc", "lewis$mapmcm"]
+  end
```

Comment:
looks ok to me. We might need to change the list of the map locations and keep only some of them; but this will be decided after we meet with the map team. I think for now it is ok to merge this.